packages/cozy-mespapiers-lib/src/components/Home/HomeToolbar.jsx

```diff
@@ -13,14 +14,14 @@ import { useMultiSelection } from '../Hooks/useMultiSelection'
 const HomeToolbar = () => {
   const { t } = useI18n()
   const { isDesktop } = useBreakpoints()
-  const history = useHistory()
+  const navigate = useNavigate()
   const { setIsMultiSelectionActive } = useMultiSelection()
   const { BarRight, BarLeft, BarCenter } = cozy.bar
 
   return (
     <>
       <BarLeft>
-        <IconButton onClick={() => history.goBack()}>
+        <IconButton onClick={() => navigate(-1)}>
```

Comment:
Can we do without -1 ? With a specific route?

Reply:
Cela arrivera sur une prochaine PR 👍